tests/test_intercept.js

```diff
@@ -178,15 +178,17 @@ test('reply takes a callback for status code', async t => {
 test('reply should throw on error on the callback', t => {
   let dataCalled = false
 
-  const scope = nock('http://www.google.com')
+  const scope = nock('http://example.com')
     .get('/')
     .reply(500, (path, requestBody, callback) =>
       callback(new Error('Database failed'))
     )
 
+  // TODO When this request is converted to `got`, it causes the request not
+  // to match.
```

Comment:
Interesting! I wonder why 🤔

Reply:
It's very strange! At first I thought perhaps the new test exposed a behavior we didn't know about, though the old test does seem to be working fine.

I wonder if there is an interop issue with got. Could it be retrying automatically?